packages/utils/scss/common/_reset.scss

```diff
@@ -0,0 +1,75 @@
+// TODO: docs
+
+@mixin kendo-utils--common--reset() {
+
+    @if $kendo-reset {
```

Comment:
I think this should be in the core package, not the utils.

Reply:
Hmm, indeed, we should make sure to move this to the Core package, as there are arbitrary styles. @magdalenaan could you extract this in a separate PR perhaps, so that we can move forward with this one ?

Reply:
resolved by extractring reset styles in a separate PR